P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+          <gi>figure</gi>. The corresponding Unicode character can then be recorded in the
+          <gi>desc</gi> element by the encoder if desired.</p>
+      <p>For example, the source text: <q>... ich überlege noch &#x1F648;</q> (English: <q>... I'm
+          still thinking &#x1F648;</q>) might be encoded as follows: <egXML
+          xmlns="http://www.tei-c.org/ns/Examples" xml:lang="de">
+          <post modality="written" generatedBy="human" synch="#lscLB.t004" who="#lscLB.A03"
+            xml:id="cmc_post22"> ... ich überlege noch <figure type="emoji" generatedBy="template">
+              <graphic url="fig1.png"/>
+              <desc type="gloss" xml:lang="en">see no evil monkey</desc>
+              <desc type="unicode">U+1F648</desc>
+            </figure>
+          </post>
+        </egXML></p>
+    </div>
+    <div xml:id="CMCgraphic">
+      <head>Posts with Graphics</head>
+      <p>A post in a CMC interaction may contain a graphic in addition to some text or even contain
+        only a graphic (without any text). As explained in <ptr target="#CMCcmcpostatts"/>, the
+        modality of such a post should be considered as <val>written</val>. To encode the graphic
+        information, the TEI element <gi>figure</gi> may be used in the appropriate place. <egXML
+          xmlns="http://www.tei-c.org/ns/Examples" source="#UND">
+          <post modality="written" generatedBy="human" synch="#t005" who="#A06" xml:id="cmc_post23">
+            <figure type="image" generatedBy="human">
+              <desc>screenshot of the google search for hairdresser "Pasha's Haare'm" with the
+                average google rating (4,5 of 5 stars), the address, the phone number, and the
+                opening hours. </desc>
+            </figure>
+          </post>
+        </egXML> The preceding example represents an encoding of a private chat post that contained
+        only a screenshot of a google search result for a hairdresser. A link to the graphic file
+        itself is not included because this is a text-only corpus that did not include images.</p>
+      <p>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples">
+          <post type="tweet" modality="written" generatedBy="human" synch="#tweetsbcrn18.t006"
+            xml:id="cmc_post_1043823300479258624" who="#u1" xml:lang="de">
+            <time generatedBy="system"> 16:24 </time> Bro Tri-Engel...so hab ich mir das
+            vorgestellt!!! @AndreLo79 #bcrn18 #wikidach @Heiko komm' mal Twitter! #Engel <figure
+              type="image" generatedBy="human">
+              <graphic url="https://pbs.twimg.com/media/DnxnwN9XsAEHXw2.jpg:large"/>
+            </figure>
+          </post>
+        </egXML> The preceding example represents an encoding of a tweet that contains some text
+        (including hashtags and mentions) and a graphic. The <gi>graphic</gi> element retains the
+        URL of the graphic on the web just as in the source.</p>
```

Comment:
This paragraph with a description of the example is better put before the example than after it.

Reply:
Rewritten at 58842bd72.